### PR TITLE
feat(rule): no-empty-yuidoc-code-blocks

### PR DESCRIPTION
**Pull request by knownasilya**
Prevent having an empty code block, e.g.

\`\`\`js
\`\`\`

in your yui docs

### CODE REVIEW

**Comment by knownasilya** [comment]
Next will be code blocks without a type/path

Edit, which are hard to regex. I can get valid code blocks, https://regex101.com/r/If2pDZ/1 but without a type, it matches the closing one as the starting point.

**Comment by Turbo87** [comment]
nice work, @knownasilya!

unfortunately I can't merge this without the permission of the Ember core team...

/cc @rwjblue @stefanpenner

**Comment by rwjblue** [comment]
LGTM

**Review by rwjblue — APPROVED** [review]

**Comment by Turbo87** [comment]
@rwjblue I still think we should move this repo into the `emberjs` org though...

**Comment by rwjblue** [comment]
Ya, sounds good

**Pull request merged**
